From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: "Jan Alexander Steffens (heftig)" <jan.steffens@gmail.com>
Date: Tue, 6 Apr 2021 20:26:11 +0000
Subject: [PATCH] data: Install D-Bus service file to correct location

dbus-broker was complaining:

> Service file
> `/usr/share//dbus-1/services/org.gnome.FileRoller.ArchiveManager1.service`
> is not named after the D-Bus name `org.gnome.ArchiveManager1`.
---
 data/meson.build                                              | 4 ++--
 ...nager1.service.in => org.gnome.ArchiveManager1.service.in} | 0
 2 files changed, 2 insertions(+), 2 deletions(-)
 rename data/{org.gnome.FileRoller.ArchiveManager1.service.in => org.gnome.ArchiveManager1.service.in} (100%)

diff --git a/data/meson.build b/data/meson.build
index c0eaeac2..7c1588e5 100644
--- a/data/meson.build
+++ b/data/meson.build
@@ -41,8 +41,8 @@ configure_file(
 )
 
 configure_file(
-  input : 'org.gnome.FileRoller.ArchiveManager1.service.in',
-  output : 'org.gnome.FileRoller.ArchiveManager1.service',
+  input : 'org.gnome.ArchiveManager1.service.in',
+  output : 'org.gnome.ArchiveManager1.service',
   configuration : service_data,
   install : true,
   install_dir : join_paths(datadir, 'dbus-1/services')
diff --git a/data/org.gnome.FileRoller.ArchiveManager1.service.in b/data/org.gnome.ArchiveManager1.service.in
similarity index 100%
rename from data/org.gnome.FileRoller.ArchiveManager1.service.in
rename to data/org.gnome.ArchiveManager1.service.in
